Cargo.toml

```diff
@@ -1,14 +1,15 @@
 [package]
 name = "{{project-name}}"
-version = "0.0.1"
+version = "0.1.0"
 authors = ["{{authors}}"]
 edition = "2018"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
-wascc-actor = "0.7.1"
+wapc-guest = "0.4.0"
+actor-core = { git = "https://github.com/wasmCloud/actor-interfaces", branch = "main" }
```

Comment:
make sure to include `features = ["guest"]` so the appropriate resources are pulled in

Reply:
> I rescind my approval,

![good day sir](https://static.wikia.nocookie.net/ytmnd-fads/images/5/5c/6802d88e153ab306ca7679a9260cce1b2842c3967362ddaeb8b57a0af900b6d3.jpg/revision/latest/scale-to-width-down/340?cb=20191130213001)